drivers/sqlboiler-mssql/driver/override/templates_test/singleton/mssql_main_test.go.tpl

```diff
@@ -1,14 +1,15 @@
 var rgxMSSQLkey = regexp.MustCompile(`(?m)^ALTER TABLE .*ADD\s+CONSTRAINT .* FOREIGN KEY.*?.*\n?REFERENCES.*`)
 
 type mssqlTester struct {
-	dbConn     *sql.DB
-	dbName     string
-	host       string
-	user       string
-	pass       string
-	sslmode    string
-	port       int
-	testDBName string
+	dbConn       *sql.DB
+	dbName       string
+	host         string
+	user         string
+	pass         string
+	sslmode      string
+	port         int
+	testDBName   string
+	shouldCreateTestDB bool
```

Comment:
It's preferable I think to use a negated boolean variable here to avoid having to set the defaults to true. It also works better for boolean arguments when you don't have to negate them on the command line.

Some options:
- SkipCreateDB
- NoCreateDB